src/engine/renderer/tr_shader.cpp

```diff
@@ -1456,8 +1456,8 @@ static bool LoadMap( shaderStage_t *stage, const char *buffer, const int bundleI
 		}
 	}
 
-	// enable parallax if an heightmap is found
-	// and not explicitely disabled by a shader keyword
+	// tell renderer to enable parallax mapping since an heightmap is found
+	// also tell renderer to not abuse normalmap alpha channel because it's an heightmap
```

Comment:
What does it mean to abuse the alpha channel?

Reply:
https://github.com/DaemonEngine/Daemon/blob/a321153b70157e9d37dd3d2b120ff93b14a8e9f8/src/engine/renderer/glsl_source/reliefMapping_fp.glsl#L45-L58

See also extensive explanation by Fuma there:
https://github.com/DaemonEngine/Daemon/issues/183#issuecomment-473691252

I'll do a wiki page one day.

Reply:
I pasted the link with Fuma explanation in comment.